typescript/sdk/src/core/events.ts

```diff
@@ -1,5 +1,5 @@
-import type { ProcessEvent } from '@abacus-network/core/types/contracts/Inbox';
-import type { DispatchEvent } from '@abacus-network/core/types/contracts/Outbox';
+import type { ProcessEvent } from '@abacus-network/core/dist/contracts/Inbox';
```

Comment:
So i tried to a bit to fix it, but i think maybe I'm running into some weird monorepo problem? Basically there is a core/contracts folder that currently contains the solidity contracts, but then there is also the `core/dist/contracts` folder which contains the types of these contracts (and is what we are trying to access here). If I add the `exports` property in package.json, my typescript setup i believe tries to access the smart contracts then?

Reply:
With some digging, Yorke and I realized the `dist` is needed here because of the way `typechain`'s auto-gened files don't conveniently export all types, like the types for events. This is fine for now and we can look into solutions if people use these often.